solution/ui/regulations/eregs-vite/src/views/PolicyRepositorySearch.vue

```diff
@@ -96,11 +96,12 @@ const getDocList = async (requestParams = "") => {
     policyDocList.value.loading = true;
 
     try {
-        policyDocList.value.results = await getPolicyDocList({
+        const contentList = await getPolicyDocList({
             apiUrl: props.apiUrl,
             cacheResponse: !props.isAuthenticated,
             requestParams,
         });
+        policyDocList.value.results = contentList.results;
```

Comment:
this looks good to me and has been validated as working on the experimental deployment.